File: tools/SharedJavaLibrary/src/network/message/torobot/RestoreHeadStiffnessMessage.java
package network.message.torobot;

import network.message.RemoteMessageToRobot;

public class RestoreHeadStiffnessMessage extends RemoteMessageToRobot {

  public RestoreHeadStiffnessMessage() {
    super(RemoteMessageToRobot.MessageType.RESTORE_HEAD_STIFFNESS, null);
  }
}

File: robot/Comm/Messages/Remote/ToRobot/RestoreHeadStiffnessMessage.cpp
#include "RestoreHeadStiffnessMessage.h"

#include <cstring>

RestoreHeadStiffnessMessage::RestoreHeadStiffnessMessage()
  : RemoteMessageToRobot(RemoteMessageToRobot::restoreHeadStiffness, 0, NULL) {
}

RestoreHeadStiffnessMessage::~RestoreHeadStiffnessMessage() {
}
